packages/keyring-controller/src/KeyringController.ts

```diff
@@ -321,12 +313,26 @@ export type SerializedKeyring = {
   metadata?: KeyringMetadata;
 };
 
+/**
+ * Cached encryption key used to encrypt/decrypt the vault.
+ */
+type CachedEncryptionKey = {
+  /**
+   * The exported encryption key string.
+   */
+  exported: string;
```

Comment:
IMO we could find a more suitable name than `exported` here. I feel like we `key` could work no? We could change the doc to match what we expect (even if type-wise we should be ok already)

```suggestion
  /**
   * The encryption key (exported as a string).
   */
  key: string;
```

We could also go with `keyValue` but I find it less elegant.

Also, it's more of a personal opinion, feel free to disagree and resolve this suggestion if you prefer the current naming. 😄

Reply:
I wanted to make a clear distinction between an exported key and the actual encryption key, as in the rest of the code and API is very confusing (and I also found confusing to call `importKey(encryptionKey.key)`

The `exported` key is a stringified JSON representation of an encryption key, which instead is usually an instance of whatever the cipher supports (CryptoKey, SubtleCryptoKey, etc.).

Though as you said, personal opinion - I'm fine with changing it to `key` or else if you prefer. 😄

Reply:
I renamed this property to `serialized` in 9e906a5fb24b